graphql_server/flask/graphqlview.py

```diff
@@ -53,26 +56,51 @@ def __init__(self, **kwargs):
             if hasattr(self, key):
                 setattr(self, key, value)
 
-        assert isinstance(
-            self.schema, GraphQLSchema
-        ), "A Schema is required to be provided to GraphQLView."
+        assert isinstance(self.schema, GraphQLSchema), "A Schema is required to be provided to GraphQLView."
 
     def get_root_value(self):
         return self.root_value
 
     def get_context(self):
-        context = (
-            copy.copy(self.context)
-            if self.context and isinstance(self.context, MutableMapping)
-            else {}
-        )
+        context = copy.copy(self.context) if self.context and isinstance(self.context, MutableMapping) else {}
         if isinstance(context, MutableMapping) and "request" not in context:
             context.update({"request": request})
         return context
 
     def get_middleware(self):
         return self.middleware
 
+    def result_results(self, request_method, data, catch):
+        return run_http_query(
+            self.schema,
+            request_method,
+            data,
+            query_data=request.args,
+            batch_enabled=self.batch,
+            catch=catch,
+            # Execute options
+            root_value=self.get_root_value(),
+            context_value=self.get_context(),
+            middleware=self.get_middleware(),
+            run_sync=not self.enable_async,
+        )
+
+    async def resolve_results_async(self, request_method, data, catch):
+        execution_results, all_params = run_http_query(
+            self.schema,
+            request_method,
+            data,
+            query_data=request.args,
+            batch_enabled=self.batch,
+            catch=catch,
+            # Execute options
+            root_value=self.get_root_value(),
+            context_value=self.get_context(),
+            middleware=self.get_middleware(),
+            run_sync=not self.enable_async,
+        )
```

Comment:
This should be replaced by a call to previous `result_results` method, to avoid repeating code ;)